app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Admin::UsersController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
We don't need to add before action here as ApplicationController already have it. Can you check if we can remove it.